content/applications/hr/payroll/salary_attachment.rst

```diff
@@ -0,0 +1,56 @@
+========================
+Salary attachment report
+========================
+
+*Salary attachments* in Odoo refer to a portion of an employee's earnings that are earmarked for
+something, both voluntary and involuntary. These can include things like contributions to a
+retirement plan, repayment of a loan, wage garnishments, or child support.
+
+To view this report, navigate to :menuselection:`Payroll app --> Reporting --> Salary Attachment
```

Comment:
Hmm- it's a default in the runbot (version 17.0). I am wondering if you opened a runbot that had issues? Here it is- it's a standard report:
![image](https://github.com/user-attachments/assets/3dd252bd-40cb-485d-9837-1e3e49044a54)


I will add- if there are NO salary attachments, the report does not appear since there is no data to show.